ONSdigital/eq-survey-runner
# Browser bugs

### Changes
- various browser-related fixes
- fixed some a11y (accessibility) issues
### How to test
- `npm install`
- `npm run compile`
- check whole questionnaire in Chrome, Firefox, Safari, IE7-11, Edge, iOS Safari
### Who can test

Anyone with a couple of days free.
